src/cdk/testing/component-harness.ts

```diff
@@ -375,6 +375,31 @@ export abstract class ComponentHarness {
   }
 }
 
+
+/**
+ * Base class for component harnesses that authors should extend if they anticipate that consumers
+ * of the harness may want to access other harnesses within the component.
+ */
+export abstract class UserContentComponentHarness extends ComponentHarness
+  implements HarnessLoader {
+
+  getChildLoader(selector: string): Promise<HarnessLoader> {
```

Comment:
Can we make this class generic on `<S extends string = string>`, then use `S` as the type of selector here. This lets us be a little stricter about what we expect to be passed if we want